src/components/overview/OverviewMain.jsx

```diff
@@ -169,6 +192,84 @@ class overviewMain extends React.Component {
     }
   }
 
+  ////////////////////////////////////////////////////////////////////////////////////
+  ///////////////////////// ZOOM IMAGE CAROUSEL CODE /////////////////////////////////////
+  ////////////////////////////////////////////////////////////////////////////////////
+
+  //this MODAL function changes the displayed image when you click the arrow
+  onImageModalArrowClick(direction) {
+    //conditional logic for direction and thumbnail row changing
+    if (direction === 'left') {
+      let newImageNumber = this.state.zoomedImages.currentImage - 1;
+      let newRowNumber = Math.floor(newImageNumber / 4);
+
+      this.setState({
+        zoomedImages: {
+          currentImage: newImageNumber,
+          otherImagesInStyle: this.state.zoomedImages.otherImagesInStyle,
+          currentThumbnailRow: newRowNumber
+        }
+      });
+    } else if (direction === 'right') {
```

Comment:
This approach is similar to how I handled my arrows. Your code is very clean and understandable.